paasta_tools/marathon_serviceinit.py

```diff
@@ -165,12 +165,16 @@ def status_marathon_job(service, instance, app_id, normal_instance_count, client
     )
 
 
-def get_verbose_status_of_marathon_app(marathon_client, app, service, instance, cluster, soa_dir):
+def get_verbose_status_of_marathon_app(marathon_client, app, job_config, system_paasta_config):
     """Takes a given marathon app object and returns the verbose details
     about the tasks, times, hosts, etc"""
     output = []
     create_datetime = datetime_from_utc_to_local(isodate.parse_datetime(app.version))
-    output.append("  Marathon app ID: %s" % PaastaColors.bold(app.id))
+    url = ("%s/ui/#/apps/%2F%s" % (
```

Comment:
Can you split this into a function, something like compose_dashboard_url(base, appid)